drivers/il3829/il3829.c

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (C) 2018 Silke Hofstra
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_il3829
+ *
+ * @{
+ * @file
+ * @brief       Device driver implementation for the il3829 display controller
+ *
+ * @author      Silke Hofstra <silke@slxh.eu>
+ */
+#include <string.h>
+#include "byteorder.h"
+#include "spi_display.h"
+
+#include "il3829.h"
+#include "il3829_params.h"
+#include "il3829_internal.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+int il3829_init(il3829_t *dev, const spi_display_params_t *params,
+                uint8_t size_x, uint16_t size_y, il3829_entry_mode_t entry_mode)
+{
+    memcpy(&dev->params, params, sizeof(spi_display_params_t));
+    dev->size_x = size_x;
+    dev->size_y = size_y;
+    dev->entry_mode = entry_mode;
+    dev->params.dummy = false;
+
+    return spi_display_init(&dev->params);
+}
+
+void il3829_display_init(il3829_t *dev)
+{
+    uint16_t y_data[2] = { 0 };
```

Comment:
The factory firmware of good-display pulls the RST pin down and up here. I would use this here too. The wake up function would make sense here then.

Reply:
See: https://github.com/RIOT-OS/RIOT/pull/12509/files#r336740648